orion/controllers/.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I know you added the `.gitignore` file last time. I think you should change it to this: `**/*/.project`

Reply:
Yeah I think my previous commits somehow got caught in this branch. The .gitignore change was not intentional.

Reply:
bump